client/packages/system/src/Patient/CreatePatientModal/PatientResultsTab.tsx

```diff
@@ -12,12 +12,39 @@ import { usePatient } from '../api';
 import { Gender, useCreatePatientStore } from '../hooks';
 import { PatientFragment } from '../api/operations.generated';
 
+function genderToGenderInput(gender: Gender): GenderInput {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const exhaustiveCheck = (_: never): GenderInput => {
+    return GenderInput.Male; // never returned
+  };
```

Comment:
I'm not seeing why the need.
Why not just default to something (unknown?) at the end?

```
    case Gender.UNKNOWN:
    default:
      return GenderInput.Unknown;
  }
```

Reply:
Its an exhaustive switch. So if I would have missed a case in the switch statement the remaining type of the `gender` variable does not has type `never` but the type of the missed enum variant and it would be a compile error.

While TypeScript is smart enough to figure that out it doesn't figured out that I already returned from all possible code path. Just to make it happy I added the `return GenderInput.Male` (but this code is not reachable).

Reply:
I see - very neat. thanks for the explanation